src/index.html

```diff
@@ -28,45 +28,367 @@ <h3 id="logo">My Logo</h3>
+              <p id="pitems">industry's standard.</p>
+            </div>
+          </div>
+          <div class="prof_div grid-element">
+            <ul class="lans">
+              <li><h4>HTML</h4></li>
+              <li><h4>Bootstrap</h4></li>
+              <li><h4>Ruby</h4></li>
+            </ul>
+          </div>
+        </div>
 
-    <section class="section">
-      <div class="container">
+        <button id="see_button" type="button">See project</button>
+      </section>
+
+      <section class="prof">
+        <div class="second-grid">
+          <div class="prof-flex grid-element">
+            <h1 id="prof_title">
+              Profesional Art <br />
+              Printing Data
+            </h1>
+            <div class="para prof-flex-items">
+              <p id="pitems">A daily selection of privately</p>
+              <p id="pitems">personalized reads; no accounts or</p>
+              <p id="pitems">sign-ups required. Has been the</p>
+              <p id="pitems">industry's standard.</p>
+            </div>
+          </div>
+          <div class="prof_div grid-element">
+            <ul class="lans">
+              <li><h4>HTML</h4></li>
+              <li><h4>Bootstrap</h4></li>
+              <li><h4>Ruby</h4></li>
+            </ul>
+          </div>
+        </div>
+
+        <button id="see_button" type="button">See project</button>
+      </section>
+      <section class="about">
         <div class="flex">
-          <h1 id="my_name">
-            Hey There.
-            <br />
-            I'm Zola.
-          </h1>
-          <h4 id="title">I'm a software Developer</h4>
-          <p id="describe_me">
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don’t hesitate to contact me.
+          <h1 id="about_me">About Me</h1>
+          <p id="describe_me_again">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
           </p>
         </div>
-        <div class="flex social">
-          <ul class="socialIcons">
-            <li class="icons">
-              <a href="#"><img src="Img/GITHUB.png" alt="Github" /></a>
-            </li>
-            <li class="icons">
-              <a href="#"><img src="Img/Linkedin.png" alt="LinkedIn" /></a>
-            </li>
-            <li class="icons">
-              <a href="#"
-                ><img src="Img/Angellist-Icon.png" alt="AngelList"
-              /></a>
+        <div class="flex resume">
+          <button id="resume" type="button">Get my resume</button>
+        </div>
+        <div id="design2" class="flex"></div>
+
+        <hr id="dash" />
+      </section>
+
+      <section class="tools">
```

Comment:
kindly, I suggest you use the `div` tag instead of the `section` tag, make sure that the `section` tag is used only on the main containers. 😄